src/encoder_network.py

```diff
@@ -1,20 +1,20 @@
-"""Encoder-Decoder Class containing Convolutional Layers."""
+"""Encoder Network Class containing Convolutional Layers."""
 
 import numpy as np
 import torch
 import torch.nn as nn
 import torchvision
 
 
-class EncoderDecoder(nn.Module):
+class EncoderNetwork(nn.Module):
```

Comment:
renaming to `EncoderNetwork` so that our two modules have clearly distinguishable names.